### PR TITLE
DASH-1231 Handle prices if no fees has been returned.

### PR DESCRIPTION
**Pull request by trukhilio**

### CODE REVIEW

File: server/src/external/fio.mjs [hunk 1]
```diff
@@ -354,46 +363,88 @@ class Fio {
...
+            .map(({ value }) => ` ${value.key}: ${value.fee}`)}`,
+        );
+
+        if (priceResults.some(({ status }) => status === 'rejected')) {
+          logger.info(
+            `FAILED PRICES FROM FIO SERVER ${priceResults
+              .filter(({ status }) => status === 'rejected')
+              .map(({ reason: { key, message } }) => ` ${key}: ${message}`)}`,
+          );
+        }
```
Comment (andreyvEze) [inline server/src/external/fio.mjs:400]: just a suggestion, maybe better to log json, not just strings with backspace

Comment (andreyvEze) [reply server/src/external/fio.mjs:400]: for both success and failed

File: server/src/jobs/orders.mjs [hunk 2]
```diff
@@ -83,7 +83,12 @@ class OrdersJob extends CommonJob {
     ) {
       fee = this.feesJson[action];
     } else {
-      fee = await fioApi.getFee(action);
+      try {
+        fee = await fioApi.getFee(action);
+      } catch (error) {
+        logger.error(`GET FEE FOR ACTION ${action} FAILED: `, error);
+      }
```
Comment (andreyvEze) [inline server/src/jobs/orders.mjs:90]: is it ok we'll return `undefined` instead of `null`?

**Review by andreyvEze — APPROVED** [review]

File: server/src/jobs/orders.mjs (hunk 2 above)
Comment (trukhilio) [reply server/src/jobs/orders.mjs:90]: Done

**Pull request merged**